### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sawravchy**
- [x] Aligning header elements to the center.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
-# Sentinel: The Sentinel of Your Microservices
-
-![CI](https://github.com/alibaba/sentinel-golang/workflows/CI/badge.svg?branch=master)
-[![Build Status](https://travis-ci.org/alibaba/sentinel-golang.svg?branch=master)](https://travis-ci.org/alibaba/sentinel-golang)
-[![codecov](https://codecov.io/gh/alibaba/sentinel-golang/branch/master/graph/badge.svg)](https://codecov.io/gh/alibaba/sentinel-golang)
-[![Go Report Card](https://goreportcard.com/badge/github.com/alibaba/sentinel-golang)](https://goreportcard.com/report/github.com/alibaba/sentinel-golang)
-[![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
-[![Gitter](https://badges.gitter.im/alibaba/Sentinel.svg)](https://gitter.im/alibaba/Sentinel)
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/9434884/43697219-3cb4ef3a-9975-11e8-9a9c-73f4f537442d.png" alt="Sentinel Logo" width="50%"/>
```
Comment (louyuting) [inline README.md:2]: From my perspective, it doesn't seem necessary to switch the document content to HTML. MD will be easier to maintain. I will revert here.
Thanks

File: README.md [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+</p>
+<h2 align="center">
+Sentinel: The Sentinel of Your Microservices
+</h2>
+
+<p align="center">
+  <a>
+    <img alt="CI" src="https://github.com/alibaba/sentinel-golang/workflows/CI/badge.svg?branch=master"></a>
```
Comment (louyuting) [inline README.md:10]: These badges are amazing, Thanks for your effort.

**Review by louyuting — APPROVED** [review]
Thanks for contribution

**Comment by sawravchy** [comment]
> Thanks for contribution

Welcome @louyuting

**Pull request merged**